packages/vitest/src/utils/env.ts

```diff
@@ -1,7 +1,6 @@
 export const isNode: boolean
   = typeof process < 'u'
   && typeof process.stdout < 'u'
-  && !process.versions?.deno
```

Comment:
I am not sure this makes sense because the variable is named `isNode`. Maybe we should add `isDeno` variable instead? Or change the condition that relies on this variable to be less runtime-specific?

Reply:
updated the PR accordingly